### PR TITLE
Fix issue with GP2 icons and Navigation Link

### PR DESCRIPTION
**Pull request by csbarbosa**
Refactor code in order to put the new GP2 icons working with the Navigation Link from react-components

### CODE REVIEW

**Review by kmaid — COMMENTED** [review]
Looks much better 👍

**Review by kmaid — APPROVED** [review]

File: packages/gp2-components/src/molecules/UserCardInfo.tsx [hunk 1]
```diff
@@ -3,9 +3,9 @@ import { crossQuery, Link, pixels } from '@asap-hub/react-components';
 import { gp2 as gp2Routing } from '@asap-hub/routing';
 import { css } from '@emotion/react';
 import { ComponentProps, Fragment } from 'react';
-import ProjectIcon from '../icons/project-icon';
+import projectIcon from '../icons/project-icon';
```
Comment (dianaguimaraes) [inline packages/gp2-components/src/molecules/UserCardInfo.tsx:6]: can we group this icon with workingGroupIcon
```suggestion
import { projectIcon, workingGroupIcon } from '../icons';
```

File: packages/gp2-components/src/molecules/UserMenu.tsx [hunk 2]
```diff
@@ -8,8 +8,9 @@ import {
   NavigationLink,
   pixels,
 } from '@asap-hub/react-components';
-import { ProjectIcon, WorkingGroupIcon } from '../icons';
+import { workingGroupIcon } from '../icons';
 import { nonMobileQuery } from '../layout';
+import projectIcon from '../icons/project-icon';
```
Comment (dianaguimaraes) [inline packages/gp2-components/src/molecules/UserMenu.tsx:13]: can we group this on line 11 instead, please?

File: packages/gp2-components/src/organisms/ProjectSummaryFooter.tsx [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { utils } from '@asap-hub/react-components';
 
 import { format, formatDistanceStrict as formatDistance } from 'date-fns';
 import dateIcon from '../icons/date-icon';
-import UsersIcon from '../icons/users-icon';
+import usersIcon from '../icons/users-icon';
```
Comment (dianaguimaraes) [inline packages/gp2-components/src/organisms/ProjectSummaryFooter.tsx:6]: same

File: packages/gp2-components/src/organisms/MainNavigation.tsx [hunk 4]
```diff
@@ -2,10 +2,10 @@ import { drawerQuery, pixels } from '@asap-hub/react-components';
...
 import { css } from '@emotion/react';
 import React from 'react';
-import DashboardIcon from '../icons/dashboard-icon';
-import ProjectIcon from '../icons/project-icon';
-import UsersIcon from '../icons/users-icon';
-import WorkingGroupIcon from '../icons/working-group-icon';
+import dashboardIcon from '../icons/dashboard-icon';
+import projectIcon from '../icons/project-icon';
+import usersIcon from '../icons/users-icon';
+import workingGroupIcon from '../icons/working-group-icon';
```
Comment (dianaguimaraes) [inline packages/gp2-components/src/organisms/MainNavigation.tsx:8]:
```suggestion
import {
  dashboardIcon,
  projectIcon,
  usersIcon,
  workingGroupIcon,
}  from '../icons';
```
can we group all the icons imports, please?

File: packages/gp2-components/src/organisms/UserWorkingGroups.tsx [hunk 5]
```diff
@@ -6,7 +6,7 @@ import { nonMobileQuery } from '../layout';
 import colors from '../templates/colors';
 import { IconWithLabel } from '../molecules';
 import userIcon from '../icons/user-icon';
-import UsersIcon from '../icons/users-icon';
+import usersIcon from '../icons/users-icon';
```
Comment (dianaguimaraes) [inline packages/gp2-components/src/organisms/UserWorkingGroups.tsx:9]: same

File: packages/gp2-components/src/organisms/WorkingGroupDetailHeader.tsx [hunk 6]
```diff
@@ -11,9 +11,8 @@ import {
 import { gp2 as gp2Routing } from '@asap-hub/routing';
 
 import { css } from '@emotion/react';
-
-import ProjectIcon from '../icons/project-icon';
-import UsersIcon from '../icons/users-icon';
+import projectIcon from '../icons/project-icon';
```
Comment (dianaguimaraes) [inline packages/gp2-components/src/organisms/WorkingGroupDetailHeader.tsx:14]: same

**Review by dianaguimaraes — APPROVED** [review]
just a minor import optimisation suggestion, but not a blocker

**Pull request merged**